src/views/Picklist-Detail.vue

```diff
@@ -3,75 +3,215 @@
+          <PicklistDetailItem :pickingList="picking.record"/>  
+        </ion-item-group>
       </ion-list>
-
-      <ion-fab vertical="bottom" horizontal="end" slot="fixed">
-        <ion-fab-button @click="presentAlertMultipleButtons">
-          <ion-icon :icon="checkmarkDone "/>
-        </ion-fab-button>
-      </ion-fab>
      </ion-content>
+
+     <ion-footer>
+      <ion-toolbar>
+        <ion-buttons class="footer-buttons">
+          <ion-button fill="outline" color="secondary" @click="scan()">
+            <ion-icon slot="start" :icon="barcodeOutline" />{{ $t("Scan") }}
+          </ion-button>
+          <ion-button  @click="completeProductPicklist" fill="outline" color="success">
+            <ion-icon slot="start" :icon="checkmarkDone" />{{ $t("Complete") }}
+          </ion-button>
+        </ion-buttons>
+      </ion-toolbar>
+    </ion-footer>
  </ion-page>
 </template>
 
 <script lang="ts">
-import { IonBackButton, IonButton, IonButtons, IonContent, IonFab, IonFabButton, IonHeader, IonIcon, IonItemDivider, IonList, IonLabel, IonPage, IonTitle, IonToolbar, alertController } from '@ionic/vue';
+import { IonBackButton, IonButton, IonButtons, IonContent,IonFooter, IonHeader, IonIcon, IonItemDivider, IonItemGroup, IonLabel, IonList, IonPage, IonTitle, IonToolbar, alertController } from '@ionic/vue';
 import { defineComponent } from 'vue';
-import { checkmarkDone } from 'ionicons/icons';
+import { barcodeOutline,checkmarkDone } from 'ionicons/icons';
 import PicklistDetailItem from '@/components/Picklist-detail-item.vue';
+import { mapGetters, useStore } from 'vuex';
+import { Plugins } from '@capacitor/core';
+import { translate } from '@/i18n'
+import { showToast } from '@/utils';
+
+const { BarcodeScanner } = Plugins;
 
 export default defineComponent({
-  name: 'Settings',
+  name: 'PicklistDetail',
   components: {
     IonBackButton, 
     IonButton, 
     IonButtons, 
-    IonContent, 
-    IonFab, 
-    IonFabButton, 
+    IonContent,
+    IonFooter, 
     IonHeader, 
-    IonIcon, 
-    IonItemDivider, 
-    IonList, 
-    IonLabel, 
+    IonIcon,
+    IonItemDivider,
+    IonItemGroup, 
+    IonLabel,
+    IonList,
     IonPage, 
     IonTitle, 
     IonToolbar,
     PicklistDetailItem
   },
-  methods: {
-      async presentAlertMultipleButtons() {
+  computed: {
+    ...mapGetters({
+      pickingList: 'picklist/getPickingList',
+      pickingItemList: 'picklist/getCurrent',
+      selectedProducts: 'picklist/getSelectedProducts',
+      getSelectedProductsToCompletePicklist: 'picklist/getSelectedProductsToCompletePicklist'
+    })
+  },
+  data() {
+    return {
+      pickingListGroup: [],
+      scanResult: '',
+      scannerActive: false
+    }
+  },
+  props: ['id'],
+  mounted () {
+    this.pickingItemList.pickingItemList.sort((a: any, b: any) => a.productName.localeCompare(b.productName, 'es', { sensitivity: 'base' }));
+    const data = this.pickingItemList.pickingItemList.reduce((r: any, e: any) => {
```

Comment:
Improve variable names and add comments what we are doing here and why?

Reply:
Done